### PR TITLE
doc on how to disable default partial response behaviour

### PR DESCRIPTION
**Pull request by homelessnessbo**
it wasn't really obvious on how to disable it

### CODE REVIEW

File: docs/components/query.md [hunk 1]
```diff
@@ -336,6 +336,7 @@ Flags:
                                  if no max_source_resolution param is specified.
       --query.partial-response   Enable partial response for queries if no
                                  partial_response param is specified.
+				 (--no-query.partial-response for disabling)
```
Comment (sylr) [inline docs/components/query.md]: Missing a space I think (at least on github view).

Comment (sylr) [reply docs/components/query.md]: Also maybe add it to the cli help output ?

Comment (homelessnessbo) [reply docs/components/query.md]: nah, everything is aligned if you go to file view mode.

Comment (homelessnessbo) [reply docs/components/query.md]: added to cli help output

Comment (homelessnessbo) [reply docs/components/query.md]: yeah, added this goddamn whitespace

File: cmd/thanos/query.go [hunk 2]
```diff
@@ -94,7 +94,7 @@ func registerQuery(m map[string]setupFunc, app *kingpin.Application) {
 	enableAutodownsampling := cmd.Flag("query.auto-downsampling", "Enable automatic adjustment (step / 5) to what source of data should be used in store gateways if no max_source_resolution param is specified.").
 		Default("false").Bool()
 
-	enablePartialResponse := cmd.Flag("query.partial-response", "Enable partial response for queries if no partial_response param is specified.").
+	enablePartialResponse := cmd.Flag("query.partial-response", "Enable partial response for queries if no partial_response param is specified. --no-query.partial-response for disabling").
```
Comment (sylr) [inline cmd/thanos/query.go]: Final dot missing.

File: docs/components/query.md [hunk 3]
```diff
@@ -336,6 +336,7 @@ Flags:
                                  if no max_source_resolution param is specified.
       --query.partial-response   Enable partial response for queries if no
                                  partial_response param is specified.
+	 			 --no-query.partial-response for disabling.
```
Comment (sylr) [inline docs/components/query.md]: Still missing a space before `--no-query.partial-response`

**Comment by homelessnessbo** [comment]
@bwplotka I don't have merge permissions, please click the button

**Pull request merged**